src/screens/Box.js

```diff
@@ -391,6 +400,15 @@ const BoxPage = () => (
             <SizesXsmallXlarge />
             <Example>"any CSS size"</Example>
           </PropertyValue>
+          <PropertyValue type="object">
+            <Description>
+              Passing an object to gap will automatially use the cssGap behavior
```

Comment:
Wording suggestion:

Passing an object to gap will automatically use the cssGap behavior. This is recommended when using `wrap` to avoid extra DOM nodes. Any t-shirt size or specific size in px, em, etc. is supported.

Reply:
updated